gh-81322: support multiple separators in Stream.readuntil (#16429)
python/cpython: Misc/NEWS.d/next/Library/2019-09-26-17-52-52.bpo-37141.onYY2-.rst

Author: Does the filename need to be changed to reflect the Github issue number instead of the BPO number?

Reviewer: I don't think so -- this is just used to make the files unique, at the next alpha release it will all be merged into a single 3.13.0a6.rst file.

> Support a list of separators in :meth:`asyncio.Stream.readuntil`, stopping

Reviewer 2: ```suggestion
Accept an iterable of separators in :meth:`asyncio.Stream.readuntil`, stopping
```

Author: Done

> Accept an iterable of separators in :meth:`asyncio.Stream.readuntil`, stopping

Reviewer 2: ```suggestion
Accept an iterable of separators in :meth:`asyncio.StreamReader.readuntil`, stopping
```
